internal/executor/instance/containerbackend/podman_linux.go

```diff
@@ -240,6 +240,10 @@ func (backend *Podman) ImageBuild(
 		}
 		q.Add("buildargs", string(jsonArgs))
 
+		if input.Pull {
```

Comment:
Is it happening for Podman only? Is there a need for Docker equivalent like `--pull` flag?

Reply:
>Is it happening for Podman only?

It's not: https://github.com/cirruslabs/cirrus-cli/pull/261/files#diff-c581abc4ab75c4bdf8b235d25334d7362af9e1521a53d55a727d0ce84ff8814eR126.